src/main/java/com/ft/methodecontentcollectionmapper/model/ContentCollectionType.java

```diff
@@ -0,0 +1,41 @@
+package com.ft.methodecontentcollectionmapper.model;
+
+import com.ft.methodecontentcollectionmapper.exception.UnsupportedTypeException;
+import com.google.common.collect.ImmutableMap;
+import javax.xml.xpath.XPathExpressionException;
+
+public enum ContentCollectionType {
+  STORY_PACKAGE_TYPE, CONTENT_PACKAGE_TYPE;
+
+  private static ImmutableMap<String, ContentCollectionType> webTypeToContentCollectionTypeMap = ImmutableMap.<String, ContentCollectionType>builder()
+      .put("editorsChoice", ContentCollectionType.STORY_PACKAGE_TYPE)
+      .put("content-package", ContentCollectionType.CONTENT_PACKAGE_TYPE)
+      .build();
+
+  private static ImmutableMap<ContentCollectionType, String> contentCollectionTypeToContentUriSuffixMap = ImmutableMap.<ContentCollectionType, String>builder()
+      .put(ContentCollectionType.STORY_PACKAGE_TYPE, "story-package")
+      .put(ContentCollectionType.CONTENT_PACKAGE_TYPE, "content-package")
+      .build();
+
+  public String getCorrespondingContentUriSuffix() {
+    return contentCollectionTypeToContentUriSuffixMap.get(this);
+  }
+
+  public static ContentCollectionType fromWebType(String webType, String uuid)
+      throws XPathExpressionException {
+    if (webTypeToContentCollectionTypeMap.get(webType) == null) {
```

Comment:
Right, thx, I'm for it, I just don't see them anymore after so much refactoring :smile: